soifunc/resize.py

```diff
@@ -14,21 +14,47 @@
 ]
 
 
+@dataclass
+class GoodScaler(GenericScaler):
```

Comment:
Might wanna also inherit `vskernels.KeepArScaler` and copy over the code for `scale` if you want to add `keep_ar` support like what we brought up on Discord.